src/__tests__/test.js

```diff
@@ -0,0 +1,38 @@
+import {SparseMatrix} from '..';
+
+describe('Sparse Matrix', () => {
+    it('mmul', () => {
+        var m1 = new SparseMatrix([[2, 0, 1], [0, 0, 3]]);
+        var m2 = new SparseMatrix([[0, 1], [2, 0], [0, 0]]);
+        var m3 = m1.mmul(m2);
+
+        expect(m1.cardinality).toEqual(3);
```

Comment:
Generally it's better to use `toBe` (it checks with ===). `toEqual` is more for objects and arrays. Doesn't have to be changed now.